### DISCUSSION TITLE
Disable editing of details, labels, and people information

### DISCUSSION THREAD

**Discussion by cryptotoad630** [photoprism/photoprism#3069] [Q&A]
Is there a way to disable editing of the metadata assigned to photos, such as details, labels, and people information?  Ideally I'm thinking permissions-based as possibly part of the upcoming in multi-user update, but in the mean time, even a switch or something at startup that would disable editing of this information would work.  The idea is to allow for public access once a usable library is ready without the risk of someone changing people assignments and other information.

**Comment by graciousgrey** [comment]
For now you can disable some actions in the UI

  https://docs.photoprism.app/user-guide/settings/general/ 

and then disable settings when starting tPhotoPrism again 

   https://docs.photoprism.app/getting-started/config-options/#feature-flags

**Reply by cryptotoad630** [reply]
Thanks.  That's what I'm doing today and works for the most part and it allows me to toggle back and forth easily enough.  It would be nice for public users to be able to see data provided in those subtabs without allowing them to modify it.  Disabling the Edit feature hides that view entirely.  Also, even with the Edit feature disabled, you can still modify the People assignments through the People view.  Just thoughts for future improvement.  Thanks.
